src/__tests__/lib/item-input/array.test.ts

```diff
@@ -454,13 +452,12 @@ describe('arrayDef', () => {
 			itemUpdateFromUserInputMock.mockResolvedValueOnce('item3') // `item3` is already used!
 			promptMock.mockResolvedValueOnce({ action: finishAction }) // from editList
 
-			expect(await def.updateFromUserInput(['item1', 'item2', 'item3'])).toStrictEqual(['item1', 'item2', 'item3'])
+			expect(await def.updateFromUserInput(['item1', 'item2', 'item3']))
+				.toStrictEqual(['item1', 'item2', 'item3'])
 
 			expect(promptMock).toHaveBeenCalledTimes(3)
-			expect(consoleLogSpy).toHaveBeenCalledTimes(1)
 			expect(consoleLogSpy).toHaveBeenCalledWith('Duplicate values are not allowed.')
```

Comment:
Should this be changed to `toHaveBeenCalledExactlyOnceWith`, like the others?

Reply:
I don't know why I had specified a number on this before. Since lots of things can log to the console, I don't like to put numbers on them.